src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+  const stylesArray = sourceString.split(';').filter(Boolean);
```

Comment:
[CODE KNOWLEDGE] - Don't need to make a copy before each operation. The `split` method does not mutate the original string, so there is no need to use `filter(Boolean)` to remove empty strings at this point. Instead, you can directly split and then handle any necessary filtering within the `forEach` loop.